packages/common/src/api/comments.ts

```diff
@@ -45,29 +46,52 @@ const commentsApi = createApi({
   endpoints: {
     // Queries
     getCommentsByTrackId: {
-      async fetch({ entityId }: { entityId: ID }, { audiusSdk }) {
+      async fetch(
+        {
+          entityId,
+          offset,
+          limit
+        }: { entityId: ID; offset?: number; limit?: number },
+        { audiusSdk }
+      ) {
         const sdk = await audiusSdk()
         const commentsRes = await sdk.tracks.trackComments({
-          trackId: encodeHashId(entityId)
+          trackId: encodeHashId(entityId),
+          offset,
+          limit
         })
-        return commentsRes?.data
+        return commentsRes?.data ?? []
       },
-      options: { type: 'query' },
+      options: { type: 'paginatedQuery' },
       async onQuerySuccess(comments: Comment[], _args, { dispatch }) {
         comments.forEach((comment) => {
           optimisticUpdateComment(comment.id, () => comment, dispatch)
         })
       }
     },
     getCommentById: {
-      async fetch({ id }: { id: string }, { audiusSdk }) {
+      async fetch({ id: _id }: { id: string }): Promise<Comment | undefined> {
+        // NOTE: we currently do not have an endpoint for this
```

Comment:
Did we remove the endpoint? Or what is 
```
sdk.comments.getComment({
  commentId: id
})
```

Reply:
Isaac replaced that endpoint with the comment replies, but I think we definitely need to rename some stuff 😅